app/controllers/phylo_trees_controller.rb

```diff
@@ -97,6 +97,12 @@ def retry
     end
   end
 
+  def download_snps
+    snp_file = Tempfile.new
+    Open3.capture3("aws", "s3", "cp", @phylo_tree.s3_outputs["SNP_annotations"], snp_file.path)
+    send_file snp_file.path, filename: "#{@phylo_tree.name.downcase.gsub(/\W/, '-')}__SNP-annotations.txt"
```

Comment:
for Open3.capture3 could you do something `unless status.success? LogUtil.log_err_and_airbrake("downloading #{@phylo_tree.s3_outputs["SNP_annotations"]} failed")`

Reply:
sure